client/my-sites/plugins/plugin-details-CTA/usps.tsx

```diff
@@ -81,7 +81,7 @@ const USPS: React.FC< Props > = ( {
 					{
 						id: 'plan',
 						className: 'title',
-						text: translate( 'Included in the Business plan (%s)', {
+						text: translate( 'Included in the Business plan (%s):', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.